OpenWeb-SampleApp/UI/Monetization/ViewModels/SocialMonetizationExampleViewModel.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SocialMonetizationExampleViewModel.swift
+//  OpenWeb-SampleApp
+//
+//  Created by Anael on 26/11/2024.
+//
+
+import Foundation
+import RxSwift
+import OpenWebSDK
+
+protocol SocialMonetizationExampleViewModelingInputs {}
+
+protocol SocialMonetizationExampleViewModelingOutputs {
+    var title: String { get }
+    var loggerViewModel: UILoggerViewModeling { get }
+    var showAdView: Observable<UIView> { get }
+}
+
+protocol SocialMonetizationExampleViewModeling {
+    var inputs: SocialMonetizationExampleViewModelingInputs { get }
+    var outputs: SocialMonetizationExampleViewModelingOutputs { get }
+}
+
+class SocialMonetizationExampleViewModel: SocialMonetizationExampleViewModeling, SocialMonetizationExampleViewModelingOutputs, SocialMonetizationExampleViewModelingInputs {
```

Comment:
Rename all classes to be with prefix: `SingleAdExample`.
Rename `socialMonetizationExampleTapped` output as well, and all other places that needs a rename.

Reply:
done